devicetypes/smartthings/fibaro-door-window-sensor.src/fibaro-door-window-sensor.groovy

```diff
@@ -105,11 +105,28 @@
 	}
 }
 
+/**
+ * Mapping of command classes and associated versions used for this DTH
+ */
+private getCommandClassVersions() {
+	[
+		0x30: 1,  // Sensor Binary
+		0x31: 2,  // Sensor MultiLevel
```

Comment:
0x31 wasn't in the CC list before but it did have a handler. It seems to make sense to add it but I'm not clear why it wasn't included before.

Reply:
@kianooshST any idea why this wasn't included before?

Reply:
No, sorry, I do not know. I can remove it, I feel like we should include it now.

Reply:
I think it's fine to leave in as long as the tests cover its usage